purchase_operating_unit/README.rst

```diff
@@ -0,0 +1,101 @@
+=================================
+Operating Unit in Purchase Orders
+=================================
+
+.. !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+   !! This file is generated by oca-gen-addon-readme !!
+   !! changes will be overwritten.                   !!
+   !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+
+.. |badge1| image:: https://img.shields.io/badge/maturity-Beta-yellow.png
+    :target: https://odoo-community.org/page/development-status
+    :alt: Beta
+.. |badge2| image:: https://img.shields.io/badge/licence-LGPL--3-blue.png
+    :target: http://www.gnu.org/licenses/lgpl-3.0-standalone.html
+    :alt: License: LGPL-3
+.. |badge3| image:: https://img.shields.io/badge/github-OCA%2Foperating--unit-lightgray.png?logo=github
+    :target: https://github.com/OCA/operating-unit/tree/12.0/purchase_operating_unit
+    :alt: OCA/operating-unit
+.. |badge4| image:: https://img.shields.io/badge/weblate-Translate%20me-F47D42.png
+    :target: https://translation.odoo-community.org/projects/operating-unit-12-0/operating-unit-12-0-purchase_operating_unit
+    :alt: Translate me on Weblate
+.. |badge5| image:: https://img.shields.io/badge/runbot-Try%20me-875A7B.png
+    :target: https://runbot.odoo-community.org/runbot/213/12.0
+    :alt: Try me on Runbot
+
+|badge1| |badge2| |badge3| |badge4| |badge5| 
+
+
+This module introduces the following features:
+- It introduces the operating unit to the purchase order.
+- The operating unit is copied to the invoice.
+- The operating unit is copied to the stock picking.
+- It implements user's security rules.
+
+**Table of contents**
+
+.. contents::
+   :local:
+
+Usage
+=====
+
+
+#. Create a PO. The Default Operating Unit is assigned to the PO. If you want,
+   you can change to another Operating Unit.
+#. Validate the PO. The Operating Unit is propagated to the Pickings and
+   Invoices.
+#. From the invoice, it is not possible to change the Operating Unit, it has to
+   be the same as the one of the PO.
+
+Known issues / Roadmap
+======================
+
```

Comment:
@sbidoul is it a side effect of the script that there are 2 lines in multiple chapter after the title?
In theory no line is needed.